plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/utils/SearchField.java

```diff
@@ -31,10 +31,16 @@
 import org.eclipse.swt.widgets.Composite;
 import org.eclipse.swt.widgets.Text;
 
+import de.ovgu.featureide.fm.ui.editors.FeatureDiagramEditor;
+import de.ovgu.featureide.fm.ui.editors.IGraphicalFeature;
+import de.ovgu.featureide.fm.ui.editors.IGraphicalFeatureModel;
+import de.ovgu.featureide.fm.ui.properties.FMPropertyManager;
+
```

Comment:
remove empty line

Reply:
doesn't work because the empty line is automaticly made.